polyfill/lib/ecmascript.mjs

```diff
@@ -2237,6 +2276,47 @@ export const ES = ObjectAssign({}, ES2020, {
       throw new RangeError(`${property} must be between ${minimum} and ${maximum}, not ${value}`);
     }
     return MathFloor(value);
+  },
+  // Overridden because the es-abstract version unconditionally uses util.inspect
+  Get: (O, P) => {
+    if (Type(O) !== 'Object') {
+      throw new TypeError('Assertion failed: Type(O) is not Object');
+    }
+    if (!ES.IsPropertyKey(P)) {
+      throw new TypeError(`Assertion failed: IsPropertyKey(P) is not true, got ${P}`);
+    }
+    return O[P];
+  },
+  LengthOfArrayLike: (obj) => {
+    if (Type(obj) !== 'Object') {
+      throw new TypeError('Assertion failed: `obj` must be an Object');
+    }
+    return ES.ToLength(ES.Get(obj, 'length'));
+  },
+  CreateListFromArrayLike: (
+    obj,
+    elementTypes = ['Undefined', 'Null', 'Boolean', 'String', 'Symbol', 'Number', 'Object']
```

Comment:
I wouldn't mind specializing to only String for the polyfill.

Reply:
I just removed the default argument since it's not called anywhere in the polyfill without the second argument.